### PR TITLE
ODC-5660-fixing linter issues in topology gherkin features

### PR DESCRIPTION
**Pull request by makambalaji**
**Description**: Fixing the lint feature file lint issues and moving the topology features to topology folder which is occurring on executing `yarn run test-cypress-devconsole-headless`
JIRA Id: https://issues.redhat.com/browse/ODC-5660

### CODE REVIEW

File: frontend/packages/dev-console/integration-tests/features/topology/topology-connectting-nodes.feature [hunk 1]
```diff
@@ -1,25 +1,26 @@
+@topology
 Feature: Connecting nodes
```
Comment (sanketpathak) [inline frontend/packages/dev-console/integration-tests/features/topology/topology-connectting-nodes.feature:2]: Can you correct the file name

Comment (makambalaji) [reply frontend/packages/dev-console/integration-tests/features/topology/topology-connectting-nodes.feature]: Renamed the file

**Pull request merged**